Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3702,15 +3702,27 @@ def get_modified_remote_data_command(client: Client, params: dict[str, str],
     if not last_update_time:
         last_update_time = remote_args.last_update
     last_update = get_time_parameter(last_update_time, epoch_format=True)
-    filter_ = f'id <= {highest_fetched_id} AND ((status!=closed AND last_persisted_time > {last_update}) OR (status=closed AND close_time > {last_update}))'  # noqa: E501
-    print_debug_msg(f'Filter to get modified offenses is: {filter_}')
+    assert isinstance(last_update, int)
+    filter_modified = f'id <= {highest_fetched_id} AND status!=closed AND last_persisted_time > {last_update}'
+    filter_closed = f'id <= {highest_fetched_id} AND status=closed AND close_time > {last_update}'
+    print_debug_msg(f'Filter to get modified offenses is: {filter_modified}')
+    print_debug_msg(f'Filter to get closed offenses is: {filter_closed}')
     # if this call fails, raise an error and stop command execution
-    offenses = client.offenses_list(range_=range_,
-                                    filter_=filter_,
-                                    sort='+last_persisted_time',
-                                    fields='id,start_time,event_count,last_persisted_time')
-    new_modified_records_ids = {str(offense.get('id')) for offense in offenses if 'id' in offense}
-    current_last_update = last_update if not offenses else int(offenses[-1].get('last_persisted_time'))
+    offenses_modified = client.offenses_list(range_=range_,
+                                             filter_=filter_modified,
+                                             sort='+last_persisted_time',
+                                             fields='id,start_time,event_count,last_persisted_time,close_time')
```

Comment:
Put the list of fields in a constant, it's the same, and in the future we might miss updating the other when we change (add/remove) fields from one of them.